packages/elvis/CHANGELOG.json

```diff
@@ -2,6 +2,19 @@
   "$schema": "../components/changelogSchema.json",
   "name": "elvis",
   "content": [
+    {
+      "version": "20.0.0? (or should it be 19.3.1?)",
```

Comment:
I guess this should be 20.0.0 as it can break due to those two custom styles you found 😞 ref https://elvia-group.slack.com/archives/G01FK9ZMP7T/p1722844238858249